### ISSUE TITLE
Clicking a daily checks another one

### ISSUE THREAD

**Issue by luminerium** [HabitRPG/habitica#5666]
I have three active dailies. I recently clicked on the bottom one and the one at the top got checked (I also got the XP and the GP of that daily), while the previous one remained unchecked. When I clicked the bottom one again, the top one got unchecked and I got the XP and GP deducted. Then I decided to click the top one, and it got checked again, normally. The middle one also checks normally.

Simply put, I have no way of checking the bottom daily, and I had to use the "repeat every" feature to keep it inactive for the moment. I'm using Chrome version 43.0.2357.134 and Windows 8. 

User ID: 5d54efc5-2e92-409c-9f4a-361fc59c8273

**Comment by Alys** [comment]
Note to devs: all task IDs are different.

@theyliveforever Please reload the page and try again. 

If that doesn't help, log out of HabitRPG, clear your browser's cache and cookies, clear local storage if you know how (don't worry yet if you don't), then close all browser windows, reopen, and log in and try again.

If even that doesn't help, we'll need you to clear your local storage. I can provide instructions if needed.

You could also try using HabitRPG on a different browser, ideally one that you've never used it on before, to see if that fixes it. This will just be a temporary measure to work out if the problem is with your Chrome browser or with your HabitRPG account.

**Comment by luminerium** [comment]
I tried again today and it worked. Thank you and sorry for bothering you!

**Comment by Alys** [comment]
No worries, thanks very much for letting us know! :)
